src/module/data/actor/base.mjs

```diff
@@ -58,13 +58,33 @@ export default class BaseActorModel extends foundry.abstract.TypeDataModel {
     };
   }
 
+  /** @override */
+  prepareBaseData() {
+    super.prepareBaseData();
+
+    this.potency = {
+      bonuses: 0
+    };
+  }
+
   /** @override */
   prepareDerivedData() {
     super.prepareDerivedData();
 
     this.stamina.winded = Math.floor(this.stamina.max / 2);
   }
 
+  /**
+   * Perform item subtype specific modifications to the actor roll data
+   * @param {object} rollData   Pointer to the roll data object after all iterable properties of this class have been assigned as a shallow copy
+   */
+  getRollData(rollData) {
```

Comment:
This might lead to confusion down the line, having a regular `getRollData` method that returns roll data and another `getRollData` method that modifies the roll data and returns nothing.

Reply:
Will rename, probably to "modifyRollData"